aws-rds-cfn-common/src/main/java/software/amazon/rds/common/handler/Commons.java

```diff
@@ -76,8 +85,16 @@ public static <M, C> ProgressEvent<M, C> handleException(
                 return ProgressEvent.failed(null, null, handlerErrorStatus.getHandlerErrorCode(), exception.getMessage());
             }
             return ProgressEvent.failed(model, context, handlerErrorStatus.getHandlerErrorCode(), exception.getMessage());
+        } else if (errorStatus instanceof UnexpectedErrorStatus) {
+            try
+            {
+                requestLogger.log("UnexpectedErrorStatus: " + getRoot_Cause_Exception_Message(rootCause) + " " + exceptionClass);
```

Comment:
Let's make null safe instead of catch i.e:
```
if(errorStatus instanceof UnexpectedErrorStatus && getRoot_Cause_Exception_Message(rootCause) != null && exceptionClass != null ) {
   requestLogger.log("UnexpectedErrorStatus: " + getRoot_Cause_Exception_Message(rootCause) + " " + exceptionClass);
}
```